app/Http/Controllers/ReportGroupController.php

```
@@ -56,7 +66,7 @@ public function show(string $id)
         return new JsonResponse([
             'success' => true,
             'message' => 'Report Group fetched successfully',
-            'data' => ReportGroupCollection::collection(ReportGroup::find($id)),
+            'data' => ReportGroup::find($id),
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\ReportGroup' in method 'show'. (undefined)

(StaticAccess)